src/clients/python/sams_client/endpoints/images.py

```diff
@@ -132,6 +132,7 @@ def generate_rendition(
         width: Optional[int] = None,
         height: Optional[int] = None,
         keep_proportions: Optional[bool] = True,
+        name: Optional[str] = None,
```

Comment:
This `name` field would also be required in the schema for `IAssetRendition`:
https://github.com/superdesk/sams/blob/develop/src/clients/python/sams_client/schemas/assets.py#L64